src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -452,6 +452,8 @@ private void WatchContainer_StatusChanged(object sender, EventArgs e)
         /// </summary>
         private void ComputeWatchContentSize(object sender, RoutedEventArgs e)
         {
+            if (IsExpanded == false || StaysOpen == false) return;
```

Comment:
MAGN-9362 Preview bubble can be blank or not expand to detail view

We will compute expanded area only in case, when preview in expanded or pinned.